test/src/layouts/landing/footer.test.tsx

```diff
@@ -1,35 +1,39 @@
 import React from 'react';
-import userEvent from '@testing-library/user-event';
+import userEvent, { UserEvent } from '@testing-library/user-event';
 import {render, screen, waitFor} from '@testing-library/preact';
 import {describe, it} from '@jest/globals';
 import { MemoryRouter } from 'react-router-dom';
 import { useContactDialog } from '~/layouts/landing/footer/flex';
 
-jest.useFakeTimers();
-const user = userEvent.setup({advanceTimers: jest.advanceTimersByTime});
-
-function ShowDialogWithParams() {
+function ShowContactDialog(props: Parameters<ReturnType<typeof useContactDialog>['ContactDialog']>[0]) {
     const {ContactDialog, open: openContactDialog} = useContactDialog();
-    const contactFormParams = [
-        { key: 'userId', value: 'test' }
-    ];
 
     return (
         <button onClick={openContactDialog}>
             Contact Us
-            <ContactDialog contactFormParams={contactFormParams} />
+            <ContactDialog {...props} />
         </button>
     );
 }
 
 describe('flex landing footer', () => {
     describe('contact dialog', () => {
+        let user: UserEvent;
```

Comment:
It's been my experience that I can set up user once, rather than needing to set it up before each test.